### DISCUSSION TITLE
Apple builds

### DISCUSSION THREAD

**Discussion by monochromec** [strawberrymusicplayer/strawberry#1032] [General]
Any chance for at least a universal build for both x86_64 and M1 any time soon? If not a native M1 build...

**Comment by jonaski** [comment]
To provide a universal package, we need to build every library that strawberry depends on universal too.
Homebrew doesn't provide universal packages and according to them it's hard: https://github.com/orgs/Homebrew/discussions/412
macports does have the ability to compile packages universal, but don't provide binaries, https://trac.macports.org/wiki/howto/buildUniversal
I have not found any CI that supports other architectures than Intel for macOS, I don't have a mac to manually build on or test on. So it's very unlikely to happen. There are also no macOS contributors for many years.
